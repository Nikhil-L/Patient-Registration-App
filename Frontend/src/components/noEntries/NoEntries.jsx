import React from "react";

/**
 * This component display no result found message 
 * on to the screen when there aren't any search results.
 */
export class NoEntries extends React.Component{

    /**
     * This method
     * @returns html code for displaying the error message.
     */
     render()
     {
         return(<h2>No result found for your search</h2>);
     }
}