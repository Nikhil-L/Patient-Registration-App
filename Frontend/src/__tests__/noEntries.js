import React from "react";
import { NoEntries } from "../components/noEntries/NoEntries";
import renderer from "react-test-renderer";

/**
 * This is to test the NoEntries component
 */
describe("App test", ()=>{
  
  /**
   * Snapshot testing
   */
  it("matches a snapshot", ()=>{
    const tree = renderer.create(<NoEntries/>).toJSON();
    expect(tree).toMatchSnapshot();
    
  })
})