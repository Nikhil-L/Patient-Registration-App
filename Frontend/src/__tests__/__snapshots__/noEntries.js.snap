// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test matches a snapshot 1`] = `
<h2>
  No result found for your search
</h2>
`;
